docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -18,6 +22,46 @@ import { SnackInline } from '~/ui/components/Snippet';
 
 <APIInstallSection />
 
+## Configuration in app.json/app.config.js
```

Comment:
```suggestion
## Configuration in app config
```

Let's change this to app config as it sometimes leads to confusion since `app.config.ts` is also a valid file format. We're also slowly introducing app config API reference pages where it mention file names.